include/deal.II/grid/grid_tools.h

```diff
@@ -295,12 +295,19 @@ namespace GridTools
    * typically, each of the vertices at the boundary of the triangulation is
    * mapped to the corresponding points in the @p new_points map.
    *
-   * The way this function works is that it solves a Laplace equation for each
+   * The unknown displacement field $u_d(\mathbf x)$ in direction $d$ is obtained from
+   * the minimization problem
+   * \f[
+   *   \rm{min}\, \frac{1}{2}\int c(\mathbf x) \mathbf \nabla u_d(\mathbf x) \cdot \mathbf \nabla u_d(\mathbf x) \,\rm d v
+   * \f]
+   * subject to prescribed constraints. The minimizer is obtained by solving the Laplace equation
    * of the dim components of a displacement field that maps the current
-   * domain into one described by @p new_points . The @p new_points array
-   * therefore represents the boundary values of this displacement field. The
-   * function then evaluates this displacement field at each vertex in the
-   * interior and uses it to place the mapped vertex where the displacement
+   * domain into one described by @p new_points . Linear finite elements with
+   * the fourth order Gaussian quadrature are used. The difference between the
```

Comment:
-> with fourth order ...

But it's not clear to me what fourth order actually refers to. Do you mean 4 quadrature points in each coordinate direction?